addons/hr_expense/static/src/views/list.xml

```diff
@@ -45,6 +45,9 @@
                 <i class="fa fa-upload fa-10x"></i>
             </div>
         </xpath>
+        <xpath expr="//div[hasclass('o_list_renderer')]" position="attributes">
+            <attribute name="t-att-class">'hr_expense h-auto o_forbidden_tooltip_parent'</attribute>
```

Comment:
t-att-class doesn't override class attribute anymore?

when you showed me I think it did not. But in a qweb view, I thought doing this would override the original class:

https://github.com/odoo/odoo/blob/1d219de36944685153a77aeeb72920b06728383e/addons/web/static/src/views/list/list_renderer.xml#L5-L6

and so you needed to do either:

```suggestion
            <attribute name="class" add="hr_expense h-auto o_forbidden_tooltip_parent" separator=" "/>
```
or:
```suggestion
            <attribute name="class">o_list_renderer o_renderer table-responsive hr_expense h-auto o_forbidden_tooltip_parent</attribute>
```